tests/correctness/resources/miniboxing/tests/compile/SI-4012-2.check

```diff
@@ -1,9 +1,3 @@
 newSource1.scala:6: error: Using named super is not supported in miniboxed classes. For a workaround, please see https://github.com/miniboxing/miniboxing-plugin/issues/166:
   super[Super].superb // okay
                ^
-newSource1.scala:7: warning: The class Super would benefit from miniboxing type parameter A, since it is instantiated by a primitive type.
```

Comment:
I thought we agreed to have that "replace `@specialized` by `@miniboxed`" warning instead. Or am I confusing the case here?